index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="stylesheets/style.cssgi">
```

Comment:
there's an error in the style link, you write _(.cssgi)_, fix it!

![Screenshot from 2021-06-25 12-46-15](https://user-images.githubusercontent.com/47992412/123406298-6e38a380-d5b3-11eb-9b3f-473f3996fd69.png)